functions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @param PDO $db the db conn
+ * @return string
+ */
+
+$servername = "localhost";
```

Comment:
functions should usually be declared before anything else, but I'm assuming these are WIP for the retrieveDb function